packages/lib/getEventTypeById.ts

```diff
@@ -0,0 +1,264 @@
+import { Prisma, PrismaClient } from "@prisma/client";
```

Comment:
this is just a common function which is used in getServersideprops and trpc end point. refer here https://github.com/calcom/cal.com/pull/5959#discussion_r1050806695 on why we are using it twice.